doc/source/install/index.rst

```diff
@@ -11,7 +11,6 @@ It is recommended to install |PXC| from official Percona software repositories
 using the corresponding package manager for your system:
 
 * :ref:`Debian or Ubuntu <apt>`
-
 * :ref:`Red Hat or CentOS <yum>`
 
 Installation Alternatives
```

Comment:
We should provide a link to the PXC Operator docs as an alternative deployment/install method.